crates/stages/src/stages/mod.rs

```diff
@@ -1,16 +1,25 @@
 /// The bodies stage.
-pub mod bodies;
```

Comment:
Reduces the length of import paths, I ran `cargo doc` a couple of times while writing this PR and it was sort of annoying to navigate 2 levels deep to find docs for a stage